client/src/components/PopupHistogram.jsx

```diff
@@ -10,7 +10,7 @@ const {
 
 const PopupHistogram = (props) => {
   const percentageArr = Object.entries(randomDistribution()).sort((x, y) => x[0] < y[0]);
-  const { rating, numReviews } = props;
+  const { rating, numReviews, boxHeight, boxWidth } = props;
```

Comment:
Why don't you just destructure the props into the parameters for your class?
Ex. PopupHistogram = ( { rating, numReviews, boxHeight, boxWidth } ) => { }